TESTING.rst

```diff
@@ -486,16 +486,57 @@ The system tests execute a specified example DAG file that runs the DAG end-to-e
 
 See more details about adding new system tests below.
 
-Running System Tests
---------------------
+Environment for System Tests
+----------------------------
+
 **Prerequisites:** You may need to set some variables to run system tests. If you need to
-add some intialization of environment variables to Breeze, you can always add a
+add some initialization of environment variables to Breeze, you can add a
 ``variables.env`` file in the ``files/airflow-breeze-config/variables.env`` file. It will be automatically
-sourced when entering the Breeze environment.
+sourced when entering the Breeze environment. You can also add some additional initialization commands
```

Comment:
Where those additional commands can be added? In `files/airflow-breeze-config/variables.env`